### PR TITLE
Change timeOut to promise and replace "childs" with "children"

### PR DESCRIPTION
**Pull request by samuelsilvapt**

### CODE REVIEW

File: calendar/index.js [hunk 1]
```diff
@@ -157,6 +157,12 @@ class IRNMNCalendar extends HTMLElement {
         this.verifyOpenDate();
         this.render();  
         this.loadFromSessionStorage();
+
+        Promise.resolve().then(() => {
```
Comment (ejcromwell) [inline calendar/index.js:161]: @samuelsilvapt I don't really understand what is happening here. You are creating a promise but you are not waiting for anything that is running async..? or am I misunderstanding what is happening

Comment (samuelsilvapt) [reply calendar/index.js:161]: @ejcromwell  an empty promise (with resolve() ) runs after the DOM updates

Comment (ejcromwell) [reply calendar/index.js:161]: I see. Would MutationObserver() be a better option. Not fussed either way, but if just using a promise, lets leave a comment to explain what DOM changes we are wairing for?

Comment (samuelsilvapt) [reply calendar/index.js:165]: I added a comment to the promise. MutationObserver() looks useful for cases like this, I wasn't familiar with

Comment (ejcromwell) [reply calendar/index.js:165]: comment works.

**Pull request merged**